requirements.txt

```diff
@@ -3,7 +3,7 @@
 argo-ams-library
 certifi<2020.4.5.2  # Used by AMS (via requests), 2020.4.5.2 dropped support for Python 2
 pyopenssl<=21.0.0  # 22.0.0 dropped support for Python 2
-cryptography==3.3.0  # Crypto dropped support for Python 2 after 3.3.  They're now on 41.X...
+cryptography==3.2.0  # Crypto dropped support for Python 2 after 3.3
```

Comment:
You could go into a bit more detail in the commit about what the compatability issues were.

Reply:
Ah, yes, it was compatibility with pyopenssl. On Python 2:

```
Traceback (most recent call last):
  File "bin/receiver.py", line 21, in <module>
    import ssm.agents
  File "/root/ssm/ssm/agents.py", line 40, in <module>
    from argo_ams_library import (AmsConnectionException, AmsTimeoutException,
  File "/usr/lib/python2.7/site-packages/argo_ams_library/__init__.py", line 11, in <module>
    from .ams import ArgoMessagingService
  File "/usr/lib/python2.7/site-packages/argo_ams_library/ams.py", line 4, in <module>
    import requests
  File "/usr/lib/python2.7/site-packages/requests/__init__.py", line 95, in <module>
    from urllib3.contrib import pyopenssl
  File "/usr/lib/python2.7/site-packages/urllib3/contrib/pyopenssl.py", line 46, in <module>
    import OpenSSL.SSL
  File "/usr/lib/python2.7/site-packages/OpenSSL/__init__.py", line 8, in <module>
    from OpenSSL import crypto, SSL
  File "/usr/lib/python2.7/site-packages/OpenSSL/SSL.py", line 664, in <module>
    _lib.Cryptography_HAS_TLSEXT_HOSTNAME, "SNI not available"
AttributeError: 'module' object has no attribute 'Cryptography_HAS_TLSEXT_HOSTNAME'
```

Reply:
Aha, we need pyopenssl>=19.1 https://github.com/pyca/pyopenssl/issues/973#issuecomment-741806876

Reply:
Sorted.